cloud-common/src/main/java/com/sequenceiq/cloudbreak/common/type/ResourceType.java

```diff
@@ -64,4 +66,8 @@
     public CommonResourceType getCommonResourceType() {
         return commonResourceType;
     }
+
+    public static boolean isInstanceResource(ResourceType resourceType) {
+        return List.of(GCP_INSTANCE, OPENSTACK_INSTANCE, MOCK_INSTANCE).contains(resourceType);
```

Comment:
make the list private static final

Reply:
This is an enum class. Moving the list to a new class is not ideal I think.

Reply:
Fixed.